<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArithmeticDialog</class>
 <widget class="QDialog" name="ArithmeticDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>代数运算</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:rgb(65, 65, 65);
}
QDialog QLabel
{
	color:rgb(221, 221, 221);
	qproperty-alignment: AlignCenter;
}
QPushButton
{
	border:2px groove rgb(221, 221, 221);
	border-radius: 6px;
	color:rgb(221, 221, 221);
	padding: 2px 5px;
}
QPushButton:hover
{
	border:2px groove rgb(221, 221, 221);
	color: rgb(83, 83, 83);
	background-color: rgb(221, 221, 221);
}
QPushButton:pressed
{
	border:2px groove rgb(70, 160, 245);
	color: rgb(221, 221, 221);
	background-color: rgb(70, 160, 245);
}
QRadioButton
{
	color:rgb(221, 221, 221);
}
</string>
  </property>
  <widget class="QPushButton" name="plusButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>加</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minusButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>减</string>
   </property>
  </widget>
  <widget class="QPushButton" name="multiplyButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>31</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>乘</string>
   </property>
  </widget>
  <widget class="QLabel" name="xPos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>X位置：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xPosValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="yPos">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lantinghei SC</family>
    </font>
   </property>
   <property name="text">
    <string>Y位置：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yPosValue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
